code/controllers/subsystem/processing/processing.dm

```diff
@@ -32,7 +32,7 @@ SUBSYSTEM_DEF(processing)
 		current_run.len--
 		if(QDELETED(thing))
 			processing -= thing
-		else if(call(thing, TYPE_PROC_REF(/datum, process))(wait * 0.1) == PROCESS_KILL)
+		else if(call(thing, TYPE_PROC_REF(/datum, process))(wait) == PROCESS_KILL)
```

Comment:
Why? From the description of the process proc, it appears that they expect the number to be "seconds", not "deciseconds".

Reply:
Looks like it might have been a mistake from when I was troubleshooting some of the processing logic early on. Good catch.